doc/prometheus_vm_microstates_collector.md

```diff
@@ -0,0 +1,37 @@
+
+
+# Module prometheus_vm_microstates_collector #
+* [Description](#description)
+
+Collects statistics about the microstate accounting of Erlang VM threads using
+[msacc](http://erlang.org/doc/man/msacc.md).
```

Comment:
```suggestion
[msacc](http://erlang.org/doc/man/msacc.html).
```

Reply:
🤔 That was the doclet changing it, I directly pasted the URL into the source.